Last fixes for v2.3.0

microhobby opened the pull request:
@andreriesco please take a look.

microhobby committed:
dotnetUno: Remove UWP project references

This is crashing the new C# Dev extension for VS Code that load the
projects from the .sln file.

Signed-off-by: Matheus Castello <matheus.castello@toradex.com>

microhobby committed:
dotnetUno: doc: Add a warning about the C# Dev Kit extension

The Uno Platform VS Code extension does not work with the C# Dev Kit
extension.

Signed-off-by: Matheus Castello <matheus.castello@toradex.com>

microhobby committed:
dotnetUno: settings: Force omnisharp use

The Uno VS Code extension only works with omnisharp, so force it to use
it.

Signed-off-by: Matheus Castello <matheus.castello@toradex.com>

microhobby committed:
dotnetUno: App.xaml.cs: remove fps counter

To be honest I do not know if this is reliable or not, but it is
definitely not needed for the final application. Let's leave
this to the Uno team.

Signed-off-by: Matheus Castello <matheus.castello@toradex.com>

diff --git a/dotnetUno/__change__.Shared/App.xaml.cs b/dotnetUno/__change__.Shared/App.xaml.cs
index 28f484e..8ebc635 100644
--- a/dotnetUno/__change__.Shared/App.xaml.cs
+++ b/dotnetUno/__change__.Shared/App.xaml.cs
@@ -40,22 +40,6 @@ namespace __change__
 #endif
         }

-        double GetCurrentFPS()
-        {
-            var newTick = DateTimeOffset.UtcNow.ToUnixTimeMilliseconds();
-            var delta = newTick - lastTick;
-            lastTick = newTick;
-
-            tickSum -= tickList[tickIndex];
-            tickSum += delta;
-            tickList[tickIndex] = delta;
-
-            if (++tickIndex == tickList.Length)
-                tickIndex = 0;
-
-            return 1000.0 / ((double)tickSum / tickList.Length);
-        }
-
         /// <summary>
         /// Invoked when the application is launched normally by the end user.  Other entry points
         /// will be used such as when the application is launched to open a specific file.
@@ -113,10 +97,6 @@ namespace __change__
             }

 #if HAS_UNO_GTK
-            GtkHost.Window.Drawn += (o, args) => {
-                Console.WriteLine($"fps: {GetCurrentFPS()}");
-            };
-
             //ApplicationView.PreferredLaunchWindowingMode = ApplicationViewWindowingMode.FullScreen;
             if (Environment.GetEnvironmentVariable("UNO_FULLSCREEN")
                 .Equals("true"))
@@ -163,7 +143,7 @@ namespace __change__
 #if DEBUG
 			// Logging is disabled by default for release builds, as it incurs a significant
 			// initialization cost from Microsoft.Extensions.Logging setup. If startup performance
-			// is a concern for your application, keep this disabled. If you're running on web or
+			// is a concern for your application, keep this disabled. If you're running on web or
 			// desktop targets, you can use url or command line parameters to enable it.
 			//
 			// For more performance documentation: https://platform.uno/docs/articles/Uno-UI-Performance.html

microhobby committed:
github: workflows: only run the workflows on dev and bookworm

Signed-off-by: Matheus Castello <matheus.castello@toradex.com>

andreriesco commented on dotnetUno/__change__.Shared/App.xaml.cs:
```diff
@@ -113,10 +97,6 @@ protected override void OnLaunched(LaunchActivatedEventArgs args)
             }
 
 #if HAS_UNO_GTK
-            GtkHost.Window.Drawn += (o, args) => {
```
This part is needed for the automated tests, as it is what is printed on the console when the application runs successfully.

microhobby replied:
good catch, I added a `Hello Torizon!` to handle the automated tests

andreriesco commented on dotnetUnoFrameBuffer/.doc/README.md:
```diff
@@ -1,4 +1,6 @@
-# .NET 6 C# Uno Platform Frame Buffer template specific documentation
+# .NET 6 C# Uno Platform Fra
```
I think here a part of the phrase has been removed: "# .NET 6 C# Uno Platform FrameBuffer template specific documentation"

microhobby committed:
dotnetUno: Add a WriteLine for handle the automated tests

The automated tests are expecting a message from the application
to know if the application is running or not. And this message
was expecting the FPS stuff that was removed from the template.

Signed-off-by: Matheus Castello <matheus.castello@toradex.com>

microhobby committed:
README: Remove root README

These are not needed anymore since we have the documentation
in the .doc folder.

Signed-off-by: Matheus Castello <matheus.castello@toradex.com>

andreriesco commented on templates.json:
```diff
@@ -104,15 +104,15 @@
                 },
                 "debuggerConsole": null,
                 "mainFile": "__change__.Shared/App.xaml.cs",
                 "mainFileBreakpointLine": 117,
```
Just a little update here to the new line 113

andreriesco commented:
LGTM
